keras_rs/src/layers/retrieval/sampling_probability_correction.py

```diff
@@ -0,0 +1,41 @@
+from typing import Any
+
+import keras
+from keras import ops
+
+from keras_rs.src import types
+from keras_rs.src.api_export import keras_rs_export
+
+
+@keras_rs_export("keras_rs.layers.SamplingProbablityCorrection")
+class SamplingProbablityCorrection(keras.layers.Layer):
```

Comment:
Oh, also, we need a `get_config()` method

Reply:
Oh true, I added `epsilon`, which wasn't in the original class, I need a `get_config` now.